mathcomp/ssreflect/Make

```diff
@@ -15,6 +15,8 @@ fingraph.v
 finset.v
 fintype.v
 generic_quotient.v
+monoid.v
+comoid.v
```

Comment:
BTW, should these files be put in ssreflect or fingroup?

Reply:
`algebra` depends on `comoid.v` (which depends on `monoid.v`) and in the near future, `fingroup` will depend on `monoid.v`. Also, I would like in a more or less near future to make `bigop.v` depend on them.

Reply:
Yes, bigop looks like a good reason to keep that in ssreflect.
The bad news is that this puts even more perf constraint: there are quite a few users depending only on ssreflect and they expect it to be fast to compile (which should be the case again, once order gets its own package).

Reply:
I do not think it really makes ssreflect slower, except for the 15s it takes to compile both files. It looks like the real slowdown is for the subsequent files that rely on them.